examples/prometheus_exporter/main.cc

```diff
@@ -0,0 +1,168 @@
+#include "opentelemetry/exporters/prometheus/prometheus_collector.h"
+#include "opentelemetry/exporters/prometheus/prometheus_exporter.h"
+#include "opentelemetry/sdk/metrics/aggregator/counter_aggregator.h"
+#include "opentelemetry/sdk/metrics/aggregator/exact_aggregator.h"
+#include "opentelemetry/sdk/metrics/aggregator/gauge_aggregator.h"
+#include "opentelemetry/sdk/metrics/aggregator/histogram_aggregator.h"
+#include "opentelemetry/sdk/metrics/aggregator/min_max_sum_count_aggregator.h"
+#include "opentelemetry/sdk/metrics/aggregator/sketch_aggregator.h"
+
+#include <chrono>
+#include <cstdlib>
+#include <iostream>
+#include <string>
+#include <thread>
+
+namespace prom_exp   = opentelemetry::exporter::prometheus;
+namespace metric_sdk = opentelemetry::sdk::metrics;
+namespace metric_api = opentelemetry::metrics;
+
```

Comment:
Please put everything (except main) into an anonymous namespace.

`namespace {` here

Reply:
Added the anonymous namespace